backoffice/src/main/java/com/wedriveu/backoffice/controller/RabbitmqListenerUpdate.java

```diff
@@ -0,0 +1,61 @@
+package com.wedriveu.backoffice.controller;
+
+import com.wedriveu.services.shared.rabbitmq.VerticleConsumer;
+import com.wedriveu.services.shared.vertx.VertxJsonMapper;
+import com.wedriveu.shared.rabbitmq.message.VehicleCounter;
+import com.wedriveu.shared.util.Constants;
+import com.wedriveu.shared.util.Log;
+import io.vertx.core.Future;
+import io.vertx.core.eventbus.Message;
+import io.vertx.core.json.JsonObject;
+
+import static com.wedriveu.shared.util.Constants.*;
+
+/**
+ * @author Stefano Bernagozzi
+ */
+public class RabbitmqListenerUpdate extends VerticleConsumer{
```

Comment:
Maybe `AnalyticsVehiclesResponseConsumer` is a better name? And from a design point of view, why is this (a `controller`) a consumer? Shouldn't be only the `boundary` that can communicate with outside world???